vpr/src/draw/ui_setup.cpp

```diff
@@ -22,7 +22,7 @@
 #    include "ezgl/point.hpp"
 #    include "ezgl/application.hpp"
 #    include "ezgl/graphics.hpp"
-
+void three_dimension_layers(GtkWidget* widget, gint /*response_id*/, gpointer /*data*/);
```

Comment:
Delete if unused.

Reply:
Don't declare functions that are non-static in the .cpp file; they should be declared in a .h.